Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -793,6 +954,7 @@ BTDiagnostics::GetZSliceInDomainFlag (const int i_buffer, const int lev)
 void
 BTDiagnostics::Flush (int i_buffer)
 {
+    amrex::Print() << " in flush buffer " << i_buffer << "\n";
```

Comment:
Leftover debug statement?
```suggestion
    // amrex::AllPrint() << " in flush buffer " << i_buffer << "\n";
```